docs/2_2_common_schema.adoc

```diff
@@ -1139,11 +1139,11 @@ Variable elements can uniformly represent variables of primitive (atomic) types,
 Variable elements representing array variables must contain a `Dimensions` element specifying the array dimensions.
 The `Dimensions` element contains a sequence of `Dimension` elements, each specifying the size of one dimension of the array:
 
-- If the `<<start>> attribute of the `Dimension` element is present, it defines a constant size for this dimension, namely the integer value of the <<start>> attribute.
+- If the <<start>> attribute of the `Dimension` element is present, it defines a constant `fmi3Uint32` size for this dimension.
```

Comment:
```suggestion
- If the <<start>> attribute of the `Dimension` element is present, it defines a constant unsigned 32-bit integer size for this dimension.
```

Again since we are talking about non-API stuff, better to not reference the API typedefs.